app/assets/images/payment_icons/cimbthai.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-cimbthai">
+    <title id="pi-cimbthai">CIMB Thai</title>
+	<style>.st2{fill:#7d0b10}</style>
+<path  d="M35 0H3a3 3 0 0 0-3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3V3a3 3 0 0 0-3-3z" opacity=".1"/>
+<path  d="M35 1a2 2 0 0 1 2 2v18a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/>
+<path class="st2" d="M5.6 9.7h4.6v5H5.6v-5zm1.6 2.5c-.2.6-.7 1-1 1.7h2.3c.2 0 .3-.1.4-.3l.7-1c.1-.1.2-.3.1-.5L8.9 11c-.1-.1-.1-.3-.3-.2H6.3l.9 1.4zM17.7 12.6l.8-1.6c.2-.4.4-.6.8-.4v3.2h-.6v-1.6c-.5.5-.7 1.1-1.1 1.7l-.9-1.7h-.1v1.6H16v-3.2c.6-.1.6-.1.8.4l.9 1.6zM22 12.2l.2.3c.3.7-.1 1.3-.8 1.3h-1.3v-3.3h1.3c.8 0 1.1.7.8 1.4l-.2.3zm-1.3 1.1c.3-.1.7.1.9-.3.1-.1.1-.3 0-.4-.3-.3-.6-.2-.9-.1v.8zm0-1.4c.3 0 .6.1.8-.1.1-.1.1-.3 0-.4-.2-.2-.5-.1-.8-.1v.6zM13.6 13.1l.4.4c-.6.6-1.5.6-2.2 0-.7-.7-.7-1.8-.1-2.5s1.5-.7 2.2-.1l-.4.5c-.3-.2-.6-.3-1-.2l-.5.4c-.3.4-.2 1 .1 1.4.5.3.9.4 1.5.1zM26.4 13.8v-3.2c.3-.1.4 0 .4.3v1h1.5v-1c0-.3.1-.4.4-.3v3.2c-.3.1-.4.1-.4-.3v-1.1h-1.5v1.2c0 .2-.1.3-.4.2zM29.1 13.8l1.1-3c0-.1.1-.2.3-.2.2 0 .3 0 .3.2l1.2 3c-.3.1-.5.1-.6-.2l-.1-.3c-.1-.3-.2-.4-.5-.3h-.2c-.7 0-.7 0-.9.6-.1.2-.2.4-.6.2zm.9-1.3h1l-.5-1.4-.5 1.4zM24.5 11h-.7c-.2 0-.3 0-.3-.2s.1-.2.2-.2h1.9c.2 0 .3 0 .3.2s-.1.2-.3.2h-.7v2.8c-.4.1-.5 0-.4-.3V11zM15.2 13.8h-.6v-3.2l.6-.1v3.3zM32.7 10.6v3.2c-.3.1-.4.1-.4-.3v-2.7c0-.2.1-.3.4-.2z"/>
```

Comment:
```suggestion
<path d="M5.6 9.7h4.6v5H5.6v-5zm1.6 2.5c-.2.6-.7 1-1 1.7h2.3c.2 0 .3-.1.4-.3l.7-1c.1-.1.2-.3.1-.5L8.9 11c-.1-.1-.1-.3-.3-.2H6.3l.9 1.4zM17.7 12.6l.8-1.6c.2-.4.4-.6.8-.4v3.2h-.6v-1.6c-.5.5-.7 1.1-1.1 1.7l-.9-1.7h-.1v1.6H16v-3.2c.6-.1.6-.1.8.4l.9 1.6zM22 12.2l.2.3c.3.7-.1 1.3-.8 1.3h-1.3v-3.3h1.3c.8 0 1.1.7.8 1.4l-.2.3zm-1.3 1.1c.3-.1.7.1.9-.3.1-.1.1-.3 0-.4-.3-.3-.6-.2-.9-.1v.8zm0-1.4c.3 0 .6.1.8-.1.1-.1.1-.3 0-.4-.2-.2-.5-.1-.8-.1v.6zM13.6 13.1l.4.4c-.6.6-1.5.6-2.2 0-.7-.7-.7-1.8-.1-2.5s1.5-.7 2.2-.1l-.4.5c-.3-.2-.6-.3-1-.2l-.5.4c-.3.4-.2 1 .1 1.4.5.3.9.4 1.5.1zM26.4 13.8v-3.2c.3-.1.4 0 .4.3v1h1.5v-1c0-.3.1-.4.4-.3v3.2c-.3.1-.4.1-.4-.3v-1.1h-1.5v1.2c0 .2-.1.3-.4.2zM29.1 13.8l1.1-3c0-.1.1-.2.3-.2.2 0 .3 0 .3.2l1.2 3c-.3.1-.5.1-.6-.2l-.1-.3c-.1-.3-.2-.4-.5-.3h-.2c-.7 0-.7 0-.9.6-.1.2-.2.4-.6.2zm.9-1.3h1l-.5-1.4-.5 1.4zM24.5 11h-.7c-.2 0-.3 0-.3-.2s.1-.2.2-.2h1.9c.2 0 .3 0 .3.2s-.1.2-.3.2h-.7v2.8c-.4.1-.5 0-.4-.3V11zM15.2 13.8h-.6v-3.2l.6-.1v3.3zM32.7 10.6v3.2c-.3.1-.4.1-.4-.3v-2.7c0-.2.1-.3.4-.2z" fill="#7d0b10"/>
```

Reply:
After finishing these 2 items. Could you optimize with [SVGO](https://jakearchibald.github.io/svgomg/)?